lib/internal/debugger/inspect_client.js

```diff
@@ -344,8 +344,20 @@ class Client extends EventEmitter {
     };
 
     return new Promise((resolve, reject) => {
-      this.once('error', reject);
-      this.once('ready', resolve);
+      const tearDown = () => {
+        this.removeListener('ready', onReady);
+        this.removeListener('error', onError);
+      };
+      const onError = (err) => {
+        tearDown();
+        reject(err);
+      };
+      const onReady = () => {
+        tearDown();
+        resolve();
+      };
+      this.on('error', onError);
+      this.on('ready', onReady);
```

Comment:
This can be replaced with `events.once('ready')`, which handles `'error'` event as well.

Reply:
Oh, I didn't realize `events.once` does that. Updated, thanks!